src/containers/pages/GenericPlansList/index.tsx

```diff
@@ -69,31 +74,73 @@ export const GenericPlansList = (props: GenericPlanListProps & RouteComponentPro
     loadData().catch(error => displayError(error));
   }, []);
 
-  const listViewData = (planList: GenericPlan[]) =>
-    planList.map(planObj => {
-      return [
-        <Link to={`${pageUrl}/${planObj.plan_id}`} key={planObj.plan_id}>
-          {planObj.plan_title}
-        </Link>,
-        planObj.plan_date,
-        planObj.plan_effective_period_start,
-        planObj.plan_effective_period_end,
-        planStatusDisplay[planObj.plan_status] || planObj.plan_status,
-      ];
-    });
+  const columns = [
+    {
+      Cell: (planObj: Cell<GenericPlan>) => {
+        const original = planObj.row.original;
+        return (
+          <Link to={`${pageUrl}/${original.plan_id}`} key={original.plan_id}>
+            {original.plan_title}
+          </Link>
+        );
+      },
+      Header: TITLE,
+      accessor: 'plan_title',
+    },
+    {
+      Header: DATE_CREATED,
+      accessor: 'plan_date',
+    },
+    {
+      Header: START_DATE,
+      accessor: 'plan_effective_period_start',
+    },
+    {
+      Header: END_DATE,
+      accessor: 'plan_effective_period_end',
+    },
+    {
+      Header: STATUS_HEADER,
+      accessor: (d: Dictionary) => planStatusDisplay[d.plan_status] || d.plan_status,
+      id: 'plan_status',
+    },
+  ];
+
+  const drillDownProps: Pick<
+    DrillDownTableProps<Dictionary>,
+    | 'columns'
+    | 'data'
+    | 'loading'
+    | 'loadingComponent'
+    | 'renderInBottomFilterBar'
+    | 'renderInTopFilterBar'
+    | 'useDrillDown'
+    | 'renderNullDataComponent'
+  > = {
```

Comment:
I have seen this used a couple of places.  Could we create this as a type alias for convenient re-use?  Not a fan of all these repeated magic strings